packages/wallets/src/utils/index.ts

```diff
@@ -0,0 +1,3 @@
+export async function sleep(ms: number) {
```

Comment:
when calling sleep, we should consider the case where api calls might timeout or be rate limited

Reply:
👍

We should improve the error handling in general (the way we handle errors and the way we expose them).

Adding to the backlog as it's non-blocking.